src/Microsoft.OData.Client/ALinq/ResourceBinder.cs

```diff
@@ -799,6 +799,42 @@ internal static Expression ApplyOrdering(MethodCallExpression mce, bool descendi
             return input;
         }
 
+        internal static Expression ApplyAggregation(MethodCallExpression mce, OData.UriParser.Aggregation.AggregationMethod aggregationMethod)

```

Comment:
methodCallExpression  maybe ? The full name may be easier to read.

Reply:
@wachugamaina Fixed